File: server/php/src/Application/Domain/Payments/PaymentDetails/PaymentProduct5402SpecificOutput.php
<?php

namespace OnlinePayments\ExampleApp\Application\Domain\Payments\PaymentDetails;

final class PaymentProduct5402SpecificOutput
{
    public ?string $brand = null;

    public function __construct(?string $brand = null)
    {
        $this->brand = $brand;
    }

    public function toArray(): array
    {
        return [
            'brand' => $this->brand,
        ];
    }
}

File: server/ruby/app/business/domain/payments/payment_details/payment_product5402_specific_output.rb
module Business
  module Domain
    module Payments
      module PaymentDetails
        class PaymentProduct5402SpecificOutput
          attr_accessor :brand

          def initialize(brand: nil)
            @brand = brand
          end

          def to_h
            {
              'brand' => @brand
            }
          end
        end
      end
    end
  end
end